### ISSUE TITLE
Can no longer update or sync

### ISSUE THREAD

**Issue by sequatchiecovecreamery** [GCalToolkit/Sheets2GCal#44]
Sheets will no longer export to calendar as of two weeks ago. After copying import data into master sheet and selecting update and attempting to sync the response we receive is that "0 events created" and "0 events updated."  Nothing has changed on our end as far as we know, everything just stopped syncing two weeks ago. Any suggestions would be appreciated. I have attached a screen capture of the issue as well.
![Capture](https://github.com/GCalToolkit/Sheets2GCal/assets/162893760/f9a82832-293c-4b27-898b-74a63c197bfb)
![Capture](https://github.com/GCalToolkit/Sheets2GCal/assets/162893760/e1b7519c-f94b-4e0c-867d-6818a850273e)

**Comment by GCalToolkit** [comment]
Hi, you can move most of the columns around but column A must be "Delete" and column B must be "Update". If you put those 2 columns back where they were originally it should work normally!

**Comment by sequatchiecovecreamery** [comment]
Thank you. I believe that was definitely part of my problem. When I made the switch on the columns it still did not work though so I determined it must be something else as well. What finally worked was Importing my Calendar to Sheets and then Deleting back-logged events from the past two years. I believe the issue had to due with having to much data in the Google Calendar that it was exceeding the limit for "Google Quota" (maybe at least for the free version of Sheet2GCal?). Once I deleted some old events, I was able to upload new Calendar events no problem.

Thank you for the input and help. It got me on the right track.
